### PR TITLE
Fix incorrect device size after device rebuild

### PR DESCRIPTION
**Pull request by borzun**
After the device is rebuilt, need to update the device size, because sometimes it might be invalid. Thus, need to listen for device rebuild callback

### CODE REVIEW

**Comment by borzun** [comment]
Cancelled jobs due to changes in the OBS API, which need to be merged first (see  stream-labs/obs-studio#187 )

**Review by summeroff — APPROVED** [review]

**Review by EddyGharbi — APPROVED** [review]

**Comment by EddyGharbi** [comment]
Changes in OBS has been merged, will merge once OBS get merged in OSN.

**Pull request merged**
